testing/sdk_hosts.py

```diff
@@ -14,7 +14,7 @@
 
 
 SYSTEM_HOST_SUFFIX = 'mesos'
-AUTOIP_HOST_SUFFIX = 'autoip.dcos.thisdcos.directory'
+AUTOIP_HOST_SUFFIX = 'na2c.dcos.fakejpmc.net'
```

Comment:
we may not want to have `jpmc` here. Also, this breaks all tests which are not using custom TLDs.